lib/registry.js

```diff
@@ -127,11 +127,12 @@ class RegistryFetcher extends Fetcher {
     }
 
     const packument = await this.packument()
+    const steps = PackageJson.normalizeSteps.filter(s => s !== '_attributes')
     const mani = await new PackageJson().fromContent(pickManifest(packument, this.spec.fetchSpec, {
       ...this.opts,
       defaultTag: this.defaultTag,
       before: this.before,
-    })).normalize().then(p => p.content)
+    })).normalize({ steps }).then(p => p.content)
```

Comment:
My last PR also updated to using normalize from package-json for reading manifests in other fetchers. Should those paths do the same thing?

Reply:
Also my last pacote PR dropped the last place we were using `read-package-json` but we are still using `read-package-json-fast` in a few places.

Will we run into similar problems with those updates? Should @npmcli/package-json get a new method called parse that does not do underscore stripping?

Reply:
Only code that was directly calling `rpj.normalize` on an existing manifest would have this.  The solution there is to call it with the `_attributes` step removed.

All the other places were using rpj and rpj-fast directly, so they won't be affected the same as this one place where we were pulling a manifest from the registry and "normalizing" it.

Reply:
I don't think we need a new method just to change the steps.  I think this solution is good enough for now, and is why the steps were exported like that, so they could be changed.  And changed for just that call, not changed by manipulating the singleton package in memory like `init-package-json` used to do.